specs/OracleQueries.yml

```diff
@@ -246,6 +246,21 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/registerLocation'
+  /company/{companyNumber}/update/entityData:
```

Comment:
can we call it "/overseas-entity/{companyNumber}/entityData" rather than something update? Update sounds like some sort of generic put.

Reply:
we can then check the OE number starts with "OE" and throw an exception and stop others using this endpoint. I know that scope creep but probably a good idea not to make these endpoints too generic.